packages/cashier/src/pages/payment-agent/payment-agent-deposit-details/payment-agent-deposit-details.tsx

```diff
@@ -21,33 +21,22 @@ const PaymentAgentDepositDetails = ({ payment_agent }: TPaymentAgentDepositDetai
         );
     };
 
-    const PaymentAgentEmailDetails = () => {
+    const PaymentAgentTransferLimitDetails = () => {
         return (
-            <PaymentAgentDetail
-                action='mailto'
-                icon='IcEmailOutlineNew'
-                rel='noopener noreferrer'
-                target='_blank'
-                has_red_color
-                title={localize('Email')}
-            >
-                {payment_agent.email}
-            </PaymentAgentDetail>
-        );
-    };
-
-    const PaymentAgentMinimumWithdrawalDetails = () => {
-        return (
-            <PaymentAgentDetail icon='IcCashierMinimumWithdrawal' title={localize('Minimum withdrawal')}>
-                <Money amount={payment_agent.min_withdrawal || ''} currency={payment_agent.currency} show_currency />
-            </PaymentAgentDetail>
-        );
-    };
-
-    const PaymentAgentMaximumWithdrawalDetails = () => {
-        return (
-            <PaymentAgentDetail icon='IcCashierMaximumWithdrawal' title={localize('Maximum withdrawal')}>
-                <Money amount={payment_agent.max_withdrawal || ''} currency={payment_agent.currency} show_currency />
+            <PaymentAgentDetail icon='IcAccountTransfer' title={localize('Transfer limit')}>
```

Comment:
I think we can use IcCashierTransfer here.
```suggestion
            <PaymentAgentDetail icon='IcCashierTransfer' title={localize('Transfer limit')}>
```

Reply:
Can't seem to find IcCashierTransfer. Is there a different source for it?

<img width="535" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/03260cbc-ff82-4951-b14d-6d5e3e567740">